example/src/main/resources/application.properties.example

```diff
@@ -20,13 +20,19 @@ ldap.group.search.filter=uniqueMember={0}
 
 # See core module for example configuration
 spring.datasource.url=jdbc:postgresql://localhost:5432
+#spring.datasource.url=jdbc:mysql://localhost:3306
 spring.datasource.database=mms
 spring.datasource.username=mmsuser
+#spring.datasource.username=root #mysql
```

Comment:
You can create users in mysql as well, no need to use root.

Reply:
got lazy with using the mysql docker, we can take it out, it's just for convenience so i can quickly switch comments when testing different db